west.yml

```diff
@@ -43,7 +43,7 @@ manifest:
 
     - name: zephyr
       repo-path: zephyr
-      revision: 689d1edee1d57f052b1d4572d67618c0b0e2b8a4
+      revision: 632d6b9416c1c1813f3304ef29a6eda7954af3c3
```

Comment:
That's a good question what to write to these commits (and what should be expected). I've been including a short history for what changes this is pulling in from Zephyr, but this has not been a hard requirement. If I look at Zephyr conventions, it seems git commit description culture on west.yml updates is quite varied. Many "update to latest" mixed with commits that have detailed changelogs of pulled in changes. 

Oh well, I'll let others chime in, maybe I should stop making the verbose commits as well and just do "update to today's" as well. It's true anyone can check the changes from the source as well, and this can lower the barrier to submit west.yml changes (= you don't need to write a novel to submit one).

Reply:
@lyakh It's mostly manual what I do. I run "git log --pretty=oneline --abbrev-commit 494f88070f6bb909389630264e4f89c71072ae53.. 689d1edee1d57f052b1d4572d67618c0b0e2b8a4 " and then browse through the commit list and pick up commits that seem relevant to highlight. If it's a big update, I just search for keywords (audio/dai/dma/dsp/intel/nxp) and not go through the full list.

But yeah, let's go with a slimmer approach. That seems more aligned with how west.yml is managed in e.g. Zephyr and keeps the barrier-to-entry low.

Reply:
Keep in mind these SHA1s can point to literally anything, including unmerged forks. That's just how Github shared storage works.

https://lwn.net/Articles/884105/

Zephyr CI now has an "imposter commit" check:
- https://github.com/zephyrproject-rtos/zephyr/pull/75790

Short of such a check, I think every SHA1 in the west.yml should have at least a one-line comment with the name of the branch it comes from. Not just for safety and security but also for convenience.

Reply:
maybe we should add such a check

Reply:
I suspect adding a comment is several orders of magnitude less effort than automating such a check :-)